api/src/main/java/com/okta/sdk/authc/credentials/OAuth2ClientCredentials.java

```diff
@@ -0,0 +1,40 @@
+package com.okta.sdk.authc.credentials;
+
+import com.okta.commons.configcheck.ConfigurationValidator;
+
+public class OAuth2ClientCredentials implements ClientCredentials<String> {
+
+    private final String accessToken;
+
+    public OAuth2ClientCredentials(String accessToken) {
```

Comment:
You may want to pass in some sort of service that will manage the access token instead of a string (as the string will not be valid for long)

Reply:
Updated code